### DISCUSSION TITLE
Is it possible to display remote tracking branch name?

### DISCUSSION THREAD

**Discussion by suside** [jesseduffield/lazygit#3801] [Q&A] [answered]
Is it possible to display remote tracking branch name in _Local branches_  view? Like in `git branch -vv`:
```
  master        5ab2940b87 [origin/master: ahead 1, behind 21] Merge branch 'ISSUE-123' into 'master'
                             ^^^^^^^^^^^^

**Comment by stefanhaller** [comment, marked as answer]
We show this information when you expand the view (using `+`). In the normal, non-expanded view there's not enough space.

Also, for most people the names of the remote branches will be the same as the names of the local branch, so the only thing you need to see is whether a local branch has a remote branch or not, and you can tell by the presence or absence of either a green checkmark or yellow arrows.

**Comment by suside** [comment]
Thanks for the reply, `+` works for me :+1:
However on my rather typicał laptop (full hd screen) there is more than enough unused space:
![image](https://github.com/user-attachments/assets/4ddcb2ac-b9b7-4d8a-9691-3e0a59855f7d)
Would you guys accept PR? Perhaps with `if (enoughSpace) ` but I'm not sure if it will be easy to calculate :thinking:

**Reply by stefanhaller** [reply]
Making it dependent on the width of the view would actually be easy, but it would still have to be configurable, because it's not just a question of available space, but also of noise. I would very likely want to turn it off even if I had a large enough screen, since the extra information is unnecessary for me but makes the view more noisy. (I'd be curious why you want to see it; are your remote branch names typically different from the local names? Or do you just want to see which remote it's on?)

But then, I also think we should be careful about adding too many "obscure" configuration settings, because we have so many already and it makes the really useful ones harder to find. So I'd say I'm mildly opposed. This is not a blocking objection of course, I'll leave this to @jesseduffield to decide.

**Reply by suside** [reply]
> I'd be curious why you want to see it; are your remote branch names typically different from the local names?

The thing is they are typically the same but not always (hotfix on a feature branch or what not) and in that case I do want to see which remote it's on. Maybe `if ( localName != remoteName)` would be a better condition? No additional noise and important info only when needed at the same time.
